src/pages/Registration/Registration.js

```diff
@@ -0,0 +1,83 @@
+
```

Comment:
for this component you should use _Form_ component with validation (now your inputs don't have validation except _require_ attribute)